app/assets/stylesheets/templates/_stimulus.scss

```diff
@@ -1,12 +1,12 @@
 .stimulus {
   .hint {
-    color: #3D3DE1;
+    color: #03578C;
     font-size: 1.6rem;
     font-weight: 600;
     line-height: 2rem;
 
     a {
-      color: #3D3DE1;
+      color: #03578C;
```

Comment:
Obviously these were already in place without variables, but using color variables would improve maintainability here.